### ISSUE TITLE
Follow up analysis: "Export accession numbers" and "Export sequences" fails

### ISSUE THREAD

**Issue by hbarsnes** [compomics/peptide-shaker-2.0-issue-tracker#23]
```
java.lang.NullPointerException
    at eu.isas.peptideshaker.followup.FastaExport.include(FastaExport.java:79)
    at eu.isas.peptideshaker.followup.FastaExport.export(FastaExport.java:43)
    at eu.isas.peptideshaker.gui.export.FollowupPreferencesDialog$23.run(FollowupPreferencesDialog.java:1119)

or

java.lang.NullPointerException
    at eu.isas.peptideshaker.followup.FastaExport.include(FastaExport.java:84)
    at eu.isas.peptideshaker.followup.FastaExport.export(FastaExport.java:43)
    at eu.isas.peptideshaker.gui.export.FollowupPreferencesDialog$23.run(FollowupPreferencesDialog.java:1119)

or

java.lang.NullPointerException
    at eu.isas.peptideshaker.followup.FastaExport.include(FastaExport.java:89)
    at eu.isas.peptideshaker.followup.FastaExport.export(FastaExport.java:43)
    at eu.isas.peptideshaker.gui.export.FollowupPreferencesDialog$23.run(FollowupPreferencesDialog.java:1119)
```

**Comment by mvaudel** [comment]
Corrected in commit 88eea2b7f222626cf0dfd02afb6323e0001e27ca.

**Comment by hbarsnes** [comment]
I can confirm that the exports now work. I had to add the ">" at the start of the FASTA headers though (and also included an empty line in between the sequences to make the resulting file more readable).

But the FASTA exports now seems to contain both target and decoy sequences where it earlier only included the target sequences, which I think makes the most sense? Hopefully not too difficult to fix?

**Comment by mvaudel** [comment]
Solved in commit c6e7790b0d3ed266133befb08c13feea4642b53d.
